### DISCUSSION TITLE
Add drop shadow to OBC live timing

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#172] [Feature requests] [answered]
![image](https://user-images.githubusercontent.com/25685344/187163169-270c0e84-b118-4350-981f-3603ddd04c79.png)
Add drop shadows to the OBC live timing text to make it more readable when using a higher transparency background.

**Comment by jeppeolesen** [comment, marked as answer]
The onboard graphics have gone through several iterations since this request was created. Closing it due to it's age. Feel free to submit a new one if you would still like to request changes to the new layouts.
